src/headers/tomcrypt_pk.h

```diff
@@ -309,6 +338,12 @@ int  ecc_test(void);
 void ecc_sizes(int *low, int *high);
 int  ecc_get_size(ecc_key *key);
 
+int  ecc_get_set_by_name(const char* name, const ltc_ecc_set_type** dp);
+int  ecc_set_dp(const ltc_ecc_set_type *set, ecc_key *key);
```

Comment:
that one's redundant in the public API with `ecc_make_key_ex()`

Reply:
ah, or is there a use-case where you want to do the following?

```
ecc_get_set_by_name()
ecc_set_dp()
ecc_set_key(..., PK_PUBLIC, ...)
```

Reply:
Well, for example when manually importing the key from other/sources formats. This should provide the convenience of having function calls instead of manipulating the ecc_key struct directly.

Reply:
thou shalt never manipulate the `ecc_key` struct directly!

Reply:
The sequence: `ecc_get_set_by_name` + `ecc_set_dp` + `ecc_set_key` is IMO pretty handy. Consider `ecc_set_key` as a kind of "import key" function that is capable of loading raw private/public keys (e.g. bitcoin ECDSA keys uses this raw format, the curve is fixed/well-known "secp256k1").